builtins/amp-img.js

```diff
@@ -82,9 +95,7 @@ export class AmpImg extends BaseElement {
     if (this.img_) {
       return;
     }
-    if (!this.srcset_) {
```

Comment:
`srcset` creation was moved to `#initialize` (called by `#layoutCallback`) because it's a **very** expensive operation. Re https://github.com/ampproject/amphtml/pull/8361